napari/components/viewer_model.py

```diff
@@ -271,25 +269,30 @@ def _update_active_layer(self, event):
         # iteration goes backwards to find top most selected layer if any
         # if multiple layers are selected sets the active layer to None
 
+        # TODO: remove this in favor of directly setting selection.current.
```

Comment:
Yeah would be nice, can come later too

Reply:
added a comment.  I think this might be easy to just remove after we move to the new QtLayerList... will address at that point.